### ISSUE TITLE
ReflectionFunction::__construct(): Passing null to parameter #1 ($function) of type Closure|string is deprecated

### ISSUE THREAD

**Issue by benmoseleygray** [LaravelDaily/Test-Laravel-Routes#870]
I'm unsure how to get past this error on my dashboard.

When I go to /app/dashboard I get "ReflectionFunction::__construct(): Passing null to parameter #1 ($function) of type Closure|string is deprecated"

`Route::get('/dashboard', [DashboardController::class])->name('dashboard');`

**Comment by tiberiu-dascal** [comment]
This is not right. If you want to use a Single Action, don't use the [], just ControllerName::class
